packaging/windows/internal/build_cmake.bat

```diff
@@ -1,3 +1,3 @@
 @echo on
-msbuild "-p:Configuration=Release" torchvision.vcxproj
-msbuild "-p:Configuration=Release" INSTALL.vcxproj
+msbuild "-p:Configuration=Release" "-p:BuildInParallel=true" "-p:MultiProcessorCompilation=true" "-p:CL_MPCount=%1" torchvision.vcxproj -maxcpucount:%1
```

Comment:
So, here `/MP` is not the right option ? Funny that on windows we have to set 2 more flags...

Reply:
As far as I understand after reading the Microsoft docs, `MultiProcessorCompilation` == `/MP`. And there are many more flags, related to whether the parallelism is across projects or across C++ files. Frankly I can't say I'm fully convinced that this helped much. If we have anyone who is seasoned on working with msbuild, I would love to get their input.